embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/config/behavior/NetworkBehavior.kt

```diff
@@ -68,28 +69,27 @@ internal class NetworkBehavior(
      * List of domains to be limited for tracking.
      */
     fun getNetworkCallLimitsPerDomain(): Map<String, Int> {
-        return remote?.networkConfig?.domainLimits
-            ?: transformLocalDomainCfg()
-            ?: HashMap()
-    }
-
-    private fun transformLocalDomainCfg(): Map<String, Int>? {
-        val mergedLimits: MutableMap<String, Int> = HashMap()
-        for (domain in local?.networking?.domains ?: return null) {
-            if (domain.domain != null && domain.limit != null) {
-                mergedLimits[domain.domain] = domain.limit
+        val limitCeiling = getLimitCeiling()
```

Comment:
The algo is:

- Use the remote settings as a base
- For domains where there is both a local and remote entry, apply the local setting per domain if the local limit is smaller than the remote one
- For domains with only a local entry, apply the local setting or the ceiling as defined by the default limit set on the remote, which ever is smaller.

Reply:
Consider leaving as a comment in the code?

Reply:
Good idea